api/models/GameStateRow.js

```diff
@@ -0,0 +1,156 @@
+  /**
+   * GameState.js
+   * 
+   * @description :: A GameState record represents one move made by a player and the resulting game state
+   */
+
+module.exports = {
+  attributes: {
+    // FK to the games table
+    gameId: {
+      model: 'game',
+      required: true,
+    },
+    // Which player made the move (0 if p0, 1 if p1)
+    playedBy: {
+      type: 'number',
+      isIn: [0, 1],
+      required: true,
+    },
+    /**
+     * Enum for moveType:
+     * 1 - DRAW
+     * 2 - POINTS
+     * 3 - SCUTTLE
+     * 4 - FACECARD
+     * 5 - JACK
+     * 6 - UNTARGETED_ONE_OFF
+     * 7 - TARGETEDONE_OFF
+     * 8 - COUNTER
+     * 9 - RESOLVE
+     * 10 - RESOLVE_THREE
+     * 11 - RESOLVE_FOUR
+     * 12 - RESOLVE_FIVE
+     * 13 - SEVEN_POINTS
+     * 14 - SEVEN_SCUTTLE
+     * 15 - SEVEN_FACECARD
+     * 16 - SEVEN_JACK
+     * 17 - SEVEN_UNTARGETED_ONE_OFF
+     * 18 - SEVENTARGETEDONE_OFF
+     * 19 - PASS
+    */
+    moveType: {
+      type: 'number',
+      isIn: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
+      required: true,
+      columnType: 'int4',
+    },
+    // The card that was played
+    playedCardId: {
+      type: 'string',
+      allowNull: true,
+      columnType: 'text',
+    },
+    // The card that was targeted
+    targetCardId: {
+      type: 'string',
+      allowNull: true,
+      columnType: 'text',
+    },
+    // The 2nd card that is relevant to the move
+    targetCard2Id: {
+      type: 'string',
+      allowNull: true,
+      columnType: 'text',
+    },
```

Comment:
Upon further thought, rather than having `targetCard2Id` just for discarding from a 4, let's replace this with `discardedCards` which will be used by both 4's and 5's
```suggestion
    // Cards discarded for a 4 or 5
    discardedCards: {
      type: 'json',
      defaultsTo: [],
      columnType: 'text[]',
    },
```